### ISSUE TITLE
img-grids does not work when wikilink is disabled.

### ISSUE THREAD

**Issue by z4m1** [kepano/obsidian-minimal#695]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

7.3.1

### Describe the bug

I can't use img-grid when I put an image using markdown notation.

Below is an example of dropping an image with wikilink set to Enable.
```
![[../_media/Pasted image 20240209161023.png]]
![[../_media/Pasted image 20240209184355.png]]
```
![image](https://github.com/kepano/obsidian-minimal/assets/37892107/8e3b1628-4d8e-4e98-87aa-ef980a4bc994)

Below is an example of dropping an image with wikilink disabled.
```
![](../_media/Pasted%20image%2020240209161023.png)
![](../_media/Pasted%20image%2020240209184355.png)
```
![image](https://github.com/kepano/obsidian-minimal/assets/37892107/4e00365f-51c9-44a3-86d1-b1463f5337d8)

### Debug info

SYSTEM INFO:
	Obsidian version: v1.5.3
	Installer version: v1.5.3
	Operating system: Windows 10 Home 10.0.19045
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Base theme: dark
	Community theme: Minimal v7.4.7
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 2
	Plugins enabled: 1
		1: Minimal Theme Settings v7.3.1

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by jthile** [comment]
I was baffled by this for a while this morning. It didn’t occur to me that using proper Markdown would be the problem! Thanks for troubleshooting this.

**Comment by kepano** [comment]
Should be resolved in 7.5.4
